README.md

```diff
@@ -26,124 +27,141 @@ Delayed Parallel Image Reading for Microscopy Images in Python
 For full package documentation please visit
 [allencellmodeling.github.io/aicsimageio](https://allencellmodeling.github.io/aicsimageio/index.html).
 
-## Quick Start
+## Quickstart
 
 ### Full Image Reading
 ```python
 from aicsimageio import AICSImage, imread
 
 # Get an AICSImage object
-img = AICSImage("my_file.tiff")
-img.data  # returns 6D STCZYX numpy array
-img.dims  # returns string "STCZYX"
-img.shape  # returns tuple of dimension sizes in STCZYX order
-
-# Get 6D STCZYX numpy array
-data = imread("my_file.tiff")
+img = AICSImage("my_file.tiff")  # selects the first scene found
+img.data  # returns 5D TCZYX numpy array
+img.dims  # returns a Dimensions object
+img.dims.order  # returns string "TCZYX"
+img.dims.X  # returns size of X dimension
+img.shape  # returns tuple of dimension sizes in TCZYX order
+img.get_image_data("CZYX", T=0)  # returns 4D CZYX numpy array
+
+# Get the id of the current operating scene
+img.current_scene
+
+# Get a list valid scene ids
+img.scenes
+
+# Change scene
+img.set_scene(1)
```

Comment:
I personally don't have the know-how to have an iterator that updates the object as a part of the iteration currently... Happy to add it in later though. 4.0.* or 4.1, etc.

Reply:
We could certainly add it but I agree it's not a 4.0 change

Reply:
`for i in myimg.scenes` basically does that, right?

Reply:
sorta kinda. I think Jamie was suggesting a larger iterator where it does the `set_scene` update at the end of each loop.

Reply:
`for i in myimg.scenes` simply iterates over the scene_ids. I wouldn't call it a "full scene iterator"

Reply:
because it's hard to call set_scene! :-D

Reply:
I *am* worried that api users will perpetually be mistaking indices for ids because they are both ints, and calling `for i in range(len(myimg.scenes))`

Reply:
an iteratior type of api would definitely protect from that.    or `set_scene_by_id` to make it clear that you don't want to pass an index.

Reply:
Same. I think I just add a comment to the docstring that is like "NOTE: THESE IS NOT A RANGE OF INTS, THESE ARE UNIQUE IDs"

Reply:
Added a notes section to the docstring of `scenes` informing the user to not assume it is a range